galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunQueryRoute.java

```diff
@@ -125,14 +126,15 @@ private List<IRasSearchCriteria> getCriteria(QueryParameters queryParams) throws
 		String testName = queryParams.getSingleString("testname", null);
 		String bundle = queryParams.getSingleString("bundle", null);
 		List<String> result = queryParams.getResultsFromParameters(getResultNames());
```

Comment:
This method shouldn't be in the base QueryParameters class, but in some more RAS-specific class.